pyproject.toml

```diff
@@ -141,7 +141,7 @@ yapf = {version = "*", optional = true}
 
 # dephell ecosystem
 dephell-archive   = ">=0.1.5"
-dephell-discover  = ">=0.2.4"
+dephell-discover  = ">=0.2.6"
```

Comment:
dephell-discover 0.2.6 does not exist on Github, and on PyPI it is identical to 0.2.5 except for the version number.

Reply:
https://github.com/dephell/dephell/issues/154#issuecomment-501637968 I changed based on this but not yet check its avaliable on github or PyPI yet.

Besides that, `dephell` successfully installed via poetry.

Reply:
@CircleOnCircles, don't worry, it just isn't pushed on Github. Details in the issue:
https://github.com/dephell/dephell_discover/issues/4
Thank you for your reply :)